packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/create-projects-area/NewProjectCard.tsx

```diff
@@ -35,10 +36,10 @@ const useNewProjectCardStyles = makeStyles()((theme) => ({
 
 export const NewProjectCard = () => {
   const { classes } = useNewProjectCardStyles();
-
+  const { t } = useTranslation('sirius-web-application', { keyPrefix: 'newProjectCard' });
   return (
     <Button to={`/new/project`} component={RouterLink} className={classes.button} data-testid="create">
-      <CreateProjectAreaCard title="+ Blank project" description="Blank project">
+      <CreateProjectAreaCard title={'+ ' + t('title')} description={t('description')}>
```

Comment:
Either use the string literal, either you add the '+' symblol in the translation like deck did.
(If it was me I will do like deck did)